language-pack/src/main/res/values-ko/general.xml

```diff
@@ -48,8 +48,8 @@
     <!-- Library -->
     <string name="library_cleared">캐시가 삭제되었습니다.</string>
     <string name="library_importing">비트맵 가져오는 중...</string>
-    <string name="library_importing_several">%개의 보관된 비트맵을 찾았습니다. 가져오는 중...</string>
-    <string name="library_skin_importing_several">%s개의 보관된 스킨을 찾았습니다. 가져오는 중...</string>
```

Comment:
I looked at an example that does this in the English locale, and turns out it does not crash the game client:

https://github.com/osudroid/language-pack/blob/572ec7f4017a5e4a299b1276ceb97b7f67ecaff7/language-pack/src/main/res/values/menus.xml#L5

This can stay as is if it makes more sense.